app/models/concerns/users/order_manipulator.rb

```diff
@@ -14,6 +14,10 @@ def can_create_order_at?(organization)
       super_admin? || member_at?(organization)
     end
 
+    def can_cancel_order?(organization)
+      super_admin? || member_at?(organization)
```

Comment:
If you are just a member, should this be restricted by what state the order is in? It doesn't seem like a regular user should be able to cancel an order that may have already shipped for example.

Reply:
👍